cmd/bucket-handlers.go

```diff
@@ -927,6 +927,13 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 	maxParts := 1000
 	// Canonicalize the form values into http.Header.
 	formValues := make(http.Header)
+
+	// Multiple values for the same key (one map entry, longer slice) are cheaper
+	// than the same number of values for different keys (many map entries), but
+	// using a consistent per-value cost for overhead is simpler.
+	const mapEntryOverhead = 200
```

Comment:
only this can be outside for-loop, not maxMemoryBytes - it won't work otherwise.